.devcontainer/setup-dev-container.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+echo "Installing Ginkgo and Gomega..."
+go install github.com/onsi/ginkgo/v2/ginkgo
+go get github.com/onsi/gomega/...
+
+# Create a KIND cluster
+echo "Starting Minikube..."
+minikube start
+
+echo "Logging into GHCR (GitHub Container Registry)..."
+echo "$REPOSITORY_USER_TOKEN" | docker login ghcr.io -u "$REPOSITORY_USER" --password-stdin
```

Comment:
Where are these env variables being set ? :thinking:

Reply:
Secrets are available as ENV variables in codespaces